### ISSUE TITLE
add ability to accept connections from specific subnet / ip address

### ISSUE THREAD

**Issue by aitzkowi** [containous/traefik#623]
similar to the ability in nginx:

```
location / {
 # block one workstation
 deny    192.168.1.1;
 # allow anyone in 192.168.1.0/24
 allow   192.168.1.0/24;
 # drop rest of the world
 deny    all;
}
```

**Comment by xinity** [comment]
that would be awesome with digest auth already merged

**Comment by dikarel** [comment]
+1

My use case is as follows:

- Running intranet services on Docker
- Need ACME to secure subdomains via HTTPS
- Certain subdomains are confidential (e.g. admin dashboards) and should be IP-restricted

I'd like to be able to simplify administration by having everything on Traefik, as opposed to fiddling with Nginx/Apache.

**Comment by micmeister59** [comment]
Hi,

Is this planned in the near future?

Thanks,

**Comment by timoreimann** [comment]
This is tracked by #816 and currently implemented along #1332.

Please use these issues to track updates and provide feedback. Thanks.
